packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/image/ImageLoadEvent.kt

```diff
@@ -41,6 +41,7 @@ private constructor(
           ON_PROGRESS -> {
             putInt("loaded", loaded)
             putInt("total", total)
+            putDouble("progress", loaded/total.toDouble())
```

Comment:
Is this for consistency with iOS? Each prop we transfer has an overhead, so we should avoid transferring easily computed props.

Reply:
@javache  yes, it was for consistency, [current ios only has progress](https://github.com/facebook/react-native/blob/a7e9a222e74134fc5ab45b367df0e36702e94eb1/packages/react-native/ReactCommon/react/renderer/components/image/ImageEventEmitter.cpp#L24) it doesn't align with docs , i don't know what is the plan for this , should we remove from iOS and keep as current documentation?